generators/client/templates/src/main/webapp/content/css/main.css

```diff
@@ -278,6 +278,17 @@ Social Buttons
 .jh-table > tbody > tr > td {
     vertical-align: middle;
 }
+
+/* less visible sort icons */
+.glyphicon-sort, .glyphicon-sort-by-attributes, .glyphicon-sort-by-attributes-alt {
```

Comment:
Shouldn't this be applied only when the icon is inside the table header, else might affect icons used by users anywhere else